rs/nns/governance/tests/governance.rs

```diff
@@ -13008,9 +12976,16 @@ async fn distribute_rewards_load_test() {
     );
 
     // Step 3.1: Inspect neurons to make sure they have been rewarded for voting.
-    for neuron in governance.neuron_store.heap_neurons().values() {
+    for id in neuron_range {
+        let (neuron, actual_maturity) = governance
+            .neuron_store
+            .with_neuron(&NeuronId { id }, |neuron| {
+                (neuron.clone(), neuron.maturity_e8s_equivalent)
+            })
+            .expect("Neuron not found");
```

Comment:
This does not seem equivalent? Why not iterate over active neurons?

Reply:
Good catch.  I hadn't yet introduced that concept, b/c I didn't think I needed it outside of tests yet.  I'll go back and make this consistent.